recipes/dcmtk/all/conanfile.py

```diff
@@ -114,12 +115,16 @@ def package_id(self):
         del self.info.options.builtin_dictionary
         del self.info.options.external_dictionary
 
+    def validate_build(self):
+        if cross_building(self):
+            if self.settings.os == "Macos" and self.settings.arch != "x86_64":
+                raise ConanInvalidConfiguration("MacOS crossbuilding is only supported to target x86_64")
+            else:
+                # Note: other cross-building scenarios have not been tested and may also need to be marked as invalid
+                self.output.warning("Crossbuilding has not been tested and may not work")
```

Comment:
Cross-compilation is currently only possible if can_run() is true: https://github.com/valgur/conan-center-index/commit/0cb08c50529b844d6c50ab230225b3b9479a3476